src/app/workflows/workflows.component.html

```diff
@@ -17,7 +17,12 @@
   <div fxLayout="row" fxLayoutAlign="start center">
     <img src="../assets/svg/sub-nav/{{ (entryPageTitle$ | async | titlecase) === 'Services' ? 'services.svg' : 'workflow.svg' }}" />
     <span>&nbsp;{{ entryPageTitle$ | async | titlecase }}&nbsp;</span>
-    <div *ngIf="(entryPageTitle$ | async | titlecase) === 'Services'" class="beta-label">preview</div>
+    <div
+      *ngIf="(entryPageTitle$ | async | titlecase) === 'Services' || (entryPageTitle$ | async | titlecase) === 'Notebooks'"
```

Comment:
It was like this, so you don't have to change it, but it's a little fragile to rely on the display text for comparison -- if we ever tweak the title, this risks breaking.

You could add `this.entryType$ =this.sessionQuery.entryType$;` to workflow.component.ts, and then change this to `if (entryType$ | async == EntryType.Services || ....` (untested)